tool/loadbalancer/manager.go

```diff
@@ -0,0 +1,120 @@
+package loadbalancer
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	fileutils "github.com/gravitational/planet/lib/utils/file"
+	"github.com/gravitational/planet/lib/utils/systemd"
+	"github.com/gravitational/trace"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// NewManager creates a new manager for loadbalancer
+func NewManager() *Manager {
+	storageCh := make(chan string, 5)
+	return &Manager{
+		storage:   NewStorage(storageCh),
+		storageCh: storageCh,
+	}
+}
+
+// Manager manages the haproxy setting and the start of the service
+type Manager struct {
+	storage   *Storage
+	storageCh <-chan string
+}
+
+// GetStorage return storage object
+func (m *Manager) GetStorage() *Storage {
+	return m.storage
+}
+
+// Run runs manager
```

Comment:
```suggestion
// Run is the main manager loop. It periodically reconciles the service based on configuration updates.
```
How can a client (consumer) stop the manager?